pyproject.toml

```diff
@@ -93,10 +93,15 @@ pyvrp = "pyvrp.cli:main"
 line-length = 79
 
 
-[tool.isort]
-case_sensitive = true
-line_length = 79
-profile = "black"
+[tool.ruff]
+ignore-init-module-imports = true
+line-length = 79
+select = ["E", "F", "I", "NPY", "RET", "RSE", "RUF", "SLF", "SIM", "TCH"]
+
+
+[tool.ruff.isort]
+# case-sensitive = true

```

Comment:
https://github.com/astral-sh/ruff/issues/5514 is fixed; I'm waiting until it's released.